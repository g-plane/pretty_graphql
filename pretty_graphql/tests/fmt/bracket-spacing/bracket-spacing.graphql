query Query @directive(key: [1, 2, 3]) {
  field
}
