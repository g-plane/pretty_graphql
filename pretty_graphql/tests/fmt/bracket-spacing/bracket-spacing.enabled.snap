---
source: pretty_graphql/tests/fmt.rs
---
query Query @directive(key: [ 1, 2, 3 ]) {
  field
}
